blog/release-1-0-0-beta-4.md

```diff
@@ -0,0 +1,101 @@
+---
+keywords: [release, GreptimeDB, changelog, v1.0.0-beta.4]
+description: GreptimeDB v1.0.0-beta.4 Changelog
+date: 2025-12-29
+---
+
+# v1.0.0-beta.4
+
+Release date: December 29, 2025
+
+
+### 👍 Highlights
+
+This version fixes two critical issues:
+
```

Comment:
I think 

```
* fix(compaction): unify behavior of database compaction options with TTL by [@AntiTopQuark](https://github.com/AntiTopQuark) in [#7402](https://github.com/GreptimeTeam/greptimedb/pull/7402)
```

shoud be in highlights, it makes users can adujust the database-level compaction parameters.